recipes-android/android-init/android-init/android-init.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=/system/bin/init initializes the property service and launches servicemanager and logd
```

Comment:
I guess this description no longer really applies because init is already started right ? Maybe it'd be best to describe what the second stage does

Reply:
This service overrides the default android-init.service. So this is still relevant?